### PR TITLE
fix(floating-label): Add @noflip annotation to floating label

### LINKED ISSUE #2695: mdc-floating-label does not work with RTL post processors
** PLEASE READ THIS BEFORE FILING AN ISSUE **

## Bugs

Follow the template below **and include a codepen** to ensure the quickest and most accurate response to your issue.

### What MDC Web Version are you using?

HEAD

### What browser(s) is this bug affecting?

All

### What OS are you using?

Mac

### What are the steps to reproduce the bug?

Use mdc-floating-label.scss with an RTL post processor like CSS Janus, etc.
Load the input with dir="rtl" in the body.

### What is the expected behavior?

Label will look correct in RTL mode.

### What is the actual behavior?

Label is slightly out of place like so: 
image

### Any other information you believe would be useful?
I have a pull request for this.

### PR DESCRIPTION
**Pull request by TedSander**
Add @noflip annotation so that post RTL CSS processors do not change the values since they are already flipped for RTL.

Closes #2695

### CODE REVIEW

File: packages/mdc-floating-label/mdc-floating-label.scss [hunk 1]
```diff
@@ -40,8 +40,11 @@
     cursor: text;
 
     @include mdc-rtl {
+      /* @noflip */
       right: 0;
+      /* @noflip */
```
Comment (aprigogin) [inline packages/mdc-floating-label/mdc-floating-label.scss]: extra line above

File: packages/mdc-floating-label/mdc-floating-label.scss [hunk 2]
```diff
@@ -40,8 +40,11 @@
...
       left: auto;
+      /* @noflip */
```
Comment (aprigogin) [inline packages/mdc-floating-label/mdc-floating-label.scss]: extra line above

File: packages/mdc-floating-label/mdc-floating-label.scss (hunk 1 above)
Comment (TedSander) [reply packages/mdc-floating-label/mdc-floating-label.scss]: Done

File: packages/mdc-floating-label/mdc-floating-label.scss (hunk 2 above)
Comment (TedSander) [reply packages/mdc-floating-label/mdc-floating-label.scss]: Done

**Comment by TedSander** [comment]
Ping is the tree open again?

**Pull request merged**